molgenis-dataexplorer/src/main/resources/js/dataexplorer-directory.js

```diff
@@ -27,15 +27,15 @@ $.when($,
         var rsql = molgenis.dataexplorer.getRSQL()
         var humanReadable = ''
         if (rsql && googleSearch) {
-            uri = uri + rsql + '&*=q=' + googleSearch
+            uri = uri + rsql + ';*=q=' + encodeURIComponent(googleSearch)
```

Comment:
Nicely spotted, this & is for URLs and ; is for sql.
However, I don't get why the googleSearch would need re-encoding.